build/tools/Automation/Program.cs

```diff
@@ -29,6 +29,7 @@ private static async Task Main(string[] args)
                 .AddSingleton<IVersionService, VersionService>()
                 .AddSingleton<IYamlFileReaderService, YamlFileReaderService>()
                 .AddScoped<DotNet.DotNet>()
+                .AddScoped<Python.Python>()
```

Comment:
nit: how do you feel about renaming the `DotNet` and `Python` classes to something like `DotNetAutomator` or `DotNetProcessor` so that we can drop the namespace referenced as a prefix and just do `.AddScoped<DotNetAutomator>` ? (this also clears up what the responsibility of these classes)

Reply:
Makes sense to me; I got this updated.